packages/synapse-react-client/src/components/ChallengeSubmission/SubmissionCommitList.tsx

```diff
@@ -58,11 +57,10 @@ function SubmissionCommitList({
       disableColumnMenu: true,
       renderCell: params => {
         return (
-          <RadioOption
+          <Radio
```

Comment:
This is deep in an unmocked challenge portal component, so I couldn't directly test it without more effort than I want to spend on this. ChallengeTeamTable has a similar change that worked, so I feel confident in this.